extension/chrome/settings/modules/security.ts

```diff
@@ -60,19 +59,22 @@ View.run(class SecurityView extends View {
   };
 
   private renderPassPhraseOptionsIfStoredPermanently = async () => {
-    const keys = await KeyStore.get(this.acctEmail);
-    if (await this.isAnyPassPhraseStoredPermanently(keys)) {
+    if (await this.isAnyPassPhraseStoredPermanently(this.prvs)) {
       $('.forget_passphrase').css('display', '');
       $('.action_forget_pp').click(this.setHandler(() => {
         $('.forget_passphrase').css('display', 'none');
         $('.passphrase_entry_container').css('display', '');
       }));
       $('.confirm_passphrase_requirement_change').click(this.setHandler(async () => {
-        const primaryKiPP = await PassphraseStore.get(this.acctEmail, this.primaryKi!);
-        if ($('input#passphrase_entry').val() === primaryKiPP) {
-          for (const key of keys) {
-            await PassphraseStore.set('local', this.acctEmail, key, undefined);
-            await PassphraseStore.set('session', this.acctEmail, key, undefined);
+        // todo - for now checking just the most useful key. Should be checking that pp matches all
```

Comment:
don't quite understand the remark.
The most useful key is not defined here.

Reply:
misplaced comment, thanks